scripts/wasmd/deploy_nameservice.js

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
+
+/* eslint-disable @typescript-eslint/camelcase */
+const { SigningCosmWasmClient, Secp256k1Pen } = require("@cosmwasm/sdk");
+const fs = require("fs");
+
+const httpUrl = "http://localhost:1317";
+const faucet = {
+  mnemonic:
+    "economy stock theory fatal elder harbor betray wasp final emotion task crumble siren bottom lizard educate guess current outdoor pair theory focus wife stone",
+  address: "cosmos1pkptre7fdkl6gfrzlesjjvhxhlc3r4gmmk8rs6",
+};
+const unused = {
+  address: "cosmos1cjsxept9rkggzxztslae9ndgpdyt2408lk850u",
+};
+const guest = {
+  address: "cosmos17d0jcz59jf68g52vq38tuuncmwwjk42u6mcxej",
+};
+
+const initFree = {
+  name: "Free",
+};
+const initLuxury = {
+  name: "Luxury",
```

Comment:
I wonder if we should have instance titles/labels for all types of contracts as standarized meta data